integration.definition.ts

```diff
@@ -10,10 +10,10 @@ export default new IntegrationDefinition({
 	events,
 	configuration,
 	description: 'Allows two-way communication between the Lush Digital Estate and Botpress.',
-	icon: 'icon.jpg',
-	readme: 'hub.md',
+	icon: 'icon.svg',
+	//readme: 'hub.md',
```

Comment:
note: There's an issue on our end that causes the integration deploy to fail when a readme file is included, we're looking into this but for now we can just comment out this line.